Mathlib/Topology/Instances/ENat.lean

```diff
@@ -0,0 +1,63 @@
+/-
+Copyright (c) 2022 Peter Nelson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Peter Nelson
+-/
+import Mathlib.Data.ENat.Basic
+import Mathlib.Topology.Algebra.Monoid
+import Mathlib.Topology.Instances.Discrete
+
+/-!
+# Topology on extended natural numbers
+-/
+
+open Set
+
+open scoped Topology Filter
+
+namespace ENat
+
+instance : TopologicalSpace ℕ∞ := Preorder.topology ℕ∞
+
+instance : OrderTopology ℕ∞ := ⟨rfl⟩
+
+@[simp] theorem range_coe' : range ((↑) : ℕ → ℕ∞) = Iio ⊤ :=
+  WithTop.range_coe
+
+theorem embedding_coe : Embedding ((↑) : ℕ → ℕ∞) :=
+  Nat.strictMono_cast.embedding_of_ordConnected <| range_coe' ▸ ordConnected_Iio
+
+theorem openEmbedding_coe : OpenEmbedding ((↑) : ℕ → ℕ∞) :=
+  ⟨embedding_coe, range_coe' ▸ isOpen_Iio⟩
+
+theorem isOpen_singleton {x : ℕ∞} (hx : x ≠ ⊤) : IsOpen {x} := by
+  lift x to ℕ using hx
+  rw [← image_singleton, ← openEmbedding_coe.open_iff_image_open]
+  trivial
+
+theorem mem_nhds_iff {x : ℕ∞} {s : Set ℕ∞} (hx : x ≠ ⊤) : s ∈ 𝓝 x ↔ x ∈ s := by
+  rw [_root_.mem_nhds_iff]
+  exact ⟨fun ⟨_, h, _, h'⟩ ↦ h h', fun h ↦ ⟨_, singleton_subset_iff.2 h, isOpen_singleton hx, rfl⟩⟩
+
+theorem mem_nhds_coe_iff (n : ℕ) {s : Set ℕ∞} : s ∈ 𝓝 (n : ℕ∞) ↔ (n : ℕ∞) ∈ s :=
+  mem_nhds_iff (coe_ne_top _)
+
+@[simp] theorem nhds_cast_eq (n : ℕ) : 𝓝 (n : ℕ∞) = 𝓟 ({(n : ℕ∞)}) := by
```

Comment:
Should the naming have a `cast` or a `coe`? The lemma `mem_nhds_coe_iff` and the ENNReal counterpart uses `coe`.

Reply:
I thunk the correct name is
```suggestion
@[simp] theorem nhds_natCast (n : ℕ) : 𝓝 (n : ℕ∞) = 𝓟 ({(n : ℕ∞)}) := by
```